lib/rules/no-array-constructor.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
I'm struggling with this one: It's a clear AST change, but I can't imagine any side effects when switching from the constructor to the array literal...

I lean towards "suggestion" here since the change also helps avoid an edge case (redefinition of `Array` global).

Reply:
I can see this as a suggestion. We should probably be consistent with all of the "no" rules.